legal-api/tests/unit/services/test_involuntary_dissolution.py

```diff
@@ -39,15 +39,16 @@
 
 
 @pytest.mark.parametrize(
-        'test_name, eligible', [
-            ('TEST_ELIGIBLE', True),
-            ('TEST_INELIGIBLE', False)
+        'test_name, isAdminFreeze, eligible', [
+            ('TEST_ELIGIBLE', False, True),
+            ('TEST_INELIGIBLE', False, False),
+            ('TEST_INELIGIBLE_ADMIN_FREEZE', True, False)
         ]
 )
-def test_check_business_eligibility(session, test_name, eligible):
+def test_check_business_eligibility(session, test_name, isAdminFreeze, eligible):
```

Comment:
can we use the standard convention of using an underscore between word for param and variable naming?  i.e. `isAdminFreeze` -> `is_admin_freeze`

Reply:
thank you for pointing out, working on it

Reply:
fixed, please review